go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/k-cloud-labs/kinitiras
 
-go 1.17
+go 1.18
 
 require (
+	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/go-logr/logr v1.2.3
-	github.com/k-cloud-labs/pkg v0.3.0
+	github.com/k-cloud-labs/pkg v0.3.1-0.20221107074337-9d43a07930c7
```

Comment:
update it when pkg released new version

Reply:
yes, waiting for new valid version of `pkg`